src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlFullTextSearchMethodTranslator.cs

```diff
@@ -34,47 +37,45 @@
+            typeof(NpgsqlTsQuery).GetMethod(nameof(NpgsqlTsQuery.Parse), BindingFlags.Public | BindingFlags.Static);
 
-        static readonly MethodInfo _tsVectorParse = typeof(NpgsqlTsVector).GetMethod(
-            nameof(NpgsqlTsVector.Parse),
-            BindingFlags.Public | BindingFlags.Static);
+        static readonly MethodInfo TsVectorParse =
+            typeof(NpgsqlTsVector).GetMethod(nameof(NpgsqlTsVector.Parse), BindingFlags.Public | BindingFlags.Static);
 
-        static readonly IReadOnlyDictionary<string, string> _sqlNameByMethodName =
-            new Dictionary<string, string>
-            {
-                [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ArrayToTsVector)] = "array_to_tsvector",
-                [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ToTsVector)] = "to_tsvector",
-                [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.PlainToTsQuery)] = "plainto_tsquery",
-                [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.PhraseToTsQuery)] = "phraseto_tsquery",
-                [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ToTsQuery)] = "to_tsquery"
-            };
-
-        public Expression Translate(MethodCallExpression methodCallExpression)
+        static readonly IReadOnlyDictionary<string, string> SqlNameByMethodName = new Dictionary<string, string>
         {
-            if (methodCallExpression.Method == _tsQueryParse || methodCallExpression.Method == _tsVectorParse)
-                return new ExplicitCastExpression(
-                    methodCallExpression.Arguments[0],
-                    methodCallExpression.Method.ReturnType);
+            [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ArrayToTsVector)] = "array_to_tsvector",
+            [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ToTsVector)] = "to_tsvector",
+            [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.PlainToTsQuery)] = "plainto_tsquery",
+            [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.PhraseToTsQuery)] = "phraseto_tsquery",
+            [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ToTsQuery)] = "to_tsquery"
+        };
+
+        /// <inheritdoc />
+        [CanBeNull]
+        public Expression Translate(MethodCallExpression e)
+        {
+            if (e.Method == TsQueryParse || e.Method == TsVectorParse)
+                return new ExplicitCastExpression(e.Arguments[0], e.Method.ReturnType);
 
-            if (methodCallExpression.Method.DeclaringType == typeof(NpgsqlFullTextSearchDbFunctionsExtensions)
-                && _sqlNameByMethodName.TryGetValue(methodCallExpression.Method.Name, out var sqlFunctionName))
-                return new SqlFunctionExpression(
-                    sqlFunctionName,
-                    methodCallExpression.Method.ReturnType,
-                    methodCallExpression.Arguments.Skip(1));
+            if (e.Method.DeclaringType == typeof(NpgsqlFullTextSearchDbFunctionsExtensions) &&
```

Comment:
I'm not sure I like the "&&" operator at the end of the previous line. It is better for readability for it to be on the beginning of the subsequent line as it is normal for the human eye to pay more attention to the beginning of lines where the eye scan operation starts on the left.